distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -141,7 +144,29 @@ private void startManagedService(Configuration sourceConfig, Map<String, Object>
         ServiceManagerImpl.ServiceConfigurationListener listener =
                 new ServiceManagerImpl.ServiceConfigurationListener(sourceConfig.getPid());
 
-        getBundleContext().registerService(ConfigurationListener.class.getName(), listener, null);
+        BundleContext bundleContext = getBundleContext();
+
+        if (bundleContext == null) {
+            long millis = 0;
+            while (millis < MANAGED_SERVICE_TIMEOUT) {
+                try {
+                    Thread.sleep(CONFIG_UPDATE_WAIT_INTERVAL_MILLIS);
+                    millis += CONFIG_UPDATE_WAIT_INTERVAL_MILLIS;
+                } catch (InterruptedException e) {
+                    LOGGER.info("Interrupted exception while trying to sleep for bundle context",
+                            e);
+                }
+                LOGGER.info("Waiting for bundle context...{}ms", millis);
+                bundleContext = getBundleContext();
```

Comment:
Should either `break` here or add a second condition to while loop to test that this is not null otherwise you'll keep looping until timeout.